apps/cli/src/sync/clone.ts

```diff
@@ -97,10 +98,13 @@ export async function clone(session: ISession, remote?: string, path?: string, o
     );
     writeConfigs(session, '.', { siteConfig: newSiteConfig });
   } else if (siteConfig) {
-    const newSiteConfig: SiteConfig = {
+    const newSiteConfig: PartialSiteConfig = {
```

Comment:
I think that we likely want to clean up the names of these in the next pass, but that can be a refactor.

PartialSiteConfig, which is missing the ID, date_create etc, likely isn't what we want from the myst side.

Reply:
Hmm, I think we want `blocks.PartialSiteConfig` to become `myst.SiteConfig`, and then we have an extended `blocks.SiteConfigDTO` or something with `id`, etc...

Maybe worth a small rename now, so we aren't using "PartialSiteConfig" which sounds sketchy.